### PR TITLE
print error on stderr, if agent was deleted on the server

### PR DESCRIPTION
**Pull request by stiliev**
print error on stderr, if agent was deleted on the server and needs to be reconfigured.

### CODE REVIEW

**Comment by stiliev** [comment]
this should address https://github.com/Microsoft/vsts-agent/issues/119

File: src/Agent.Listener/MessageListener.cs [hunk 1]
```diff
@@ -103,6 +103,7 @@ public override void Initialize(IHostContext hostContext)
                     if (ex is TaskAgentNotFoundException)
                     {
                         Trace.Error("The agent no longer exists on the server. Stopping the agent.");
+                        _term.WriteError(StringUtil.Loc("MissingAgent"));
```
Comment (TingluoHuang) [inline src/Agent.Listener/MessageListener.cs]: should do similar thing for SessionConflictException or any FaultException.

Comment (bryanmacfarlane) [inline src/Agent.Listener/MessageListener.cs]: I think so since console out goes to the svc logs on nix.  Really helps.

Comment (stiliev) [inline src/Agent.Listener/MessageListener.cs]: Will print on the console for TaskAgentSessionConflictException the same message as in the trace: "The session for this agent already exists.". Should I try to make custom messages  for fatal exceptions: TaskAgentPoolNotFoundException, AccessDeniedException and TaskAgentNotFoundException or just print the message contained within (always English)?

Comment (stiliev) [inline src/Agent.Listener/MessageListener.cs]: added special message for TaskAgentSessionConflictException. The other fatal exceptions are printing a generic error message.

**Comment by bryanmacfarlane** [comment]
Is this ready?

**Comment by stiliev** [comment]
Waiting for approval. I think I did the changes that @TingluoHuang suggested

**Comment by bryanmacfarlane** [comment]
If you implemented the feedback, then merge

**Pull request merged**